oem/firstvoices/android/app/build.gradle

```diff
@@ -24,6 +24,7 @@ android {
         versionCode VERSION_CODE as Integer
         versionName VERSION_NAME
         buildConfigField "String", "VERSION_ENVIRONMENT", "\""+VERSION_ENVIRONMENT+"\""
+        buildConfigField "String", "VERSION_GIT_TAG", "\""+VERSION_GIT_TAG+"\""
```

Comment:
This needs to be done in android/KMAPro/kMAPro/build.gradle too

Reply:
Though in my debugger, BuildConfig.VERSION_GIT_TAG is coming up null...

Reply:
OK